api/src/main/java/io/strimzi/api/kafka/KafkaMirrorMakerList.java

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright 2018, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka;
+
+import io.fabric8.kubernetes.client.CustomResourceList;
+import io.strimzi.api.kafka.model.KafkaMirrorMaker;
+
+/**
+ * A {@code CustomResourceList<Topic>} required for using Fabric8 CRD support.
```

Comment:
It should be `CustomResourceList<KafkaMirrorMaker>`

Reply:
Good catch